Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs

```diff
@@ -517,6 +516,10 @@ public bool CancelOrderItems(string clientId, long? sellerId, string uuid, List<
                         {
                             updatedOrderItems.Add(orderItem);
                             orderItem.Status = customerCancelled ? BookingStatus.CustomerCancelled : BookingStatus.SellerCancelled;
+                            if (orderItem.Status == BookingStatus.SellerCancelled)
+                            {
+                                orderItem.CancellationMessage = "Order canceled by seller";
```

Comment:
Message creation should be on by default (for random opportunities), but off when the opportunity is created with SellerRequestedCancellationSimulateAction

Reply:
Not sure I understood, shouldn't we suppose to add some kind of message here only if seller is cancelled this orderItem?
You can check if the comment is still going to be relevant after I resolve comment above.

Reply:
Sorry i mean if they're created by SellerRequestedCancellationSimulateAction, then the Seller cancellation _should not_ include a message (when cancelled by the seller or the customer). If they are created with SellerRequestedCancellationSimulateAction, then it _should_ include a message _only when cancelled by the seller_.

Reply:
Ok got it, this will be covered with new flag includeCancellationMessage which will only be set in SellerRequestedCancellationWithMessageSimulateAction